lib/apiForSheets.js

```diff
@@ -67,33 +70,58 @@ module.exports.generateOutput = async function (rows, prNumber, prDetails) {
     allAuthors = [...authorSet];
 
     var userNamesNotSignedCLA = '';
+    var userNameNotInClaAndOnBoardingSheet = '';
     var claCheckStatus = false;
 
     for (var i = 0; i < allAuthors.length; i++) {
       var hasUserSignedCla = false;
+      var isUserinOnBoardingSheet = false;
       var userName = allAuthors[i];
 
       // Check if the commit author has signed the CLA.
-      for (var row in rows) {
-        var rowUserName = rows[row][0];
+      for (var row in rowsCla) {
+        var rowUserName = rowsCla[row][0];
         if (rowUserName.toLowerCase() === userName.toLowerCase()) {
           hasUserSignedCla = true;
           break;
         }
       }
 
+      for (var row in rowsOnBoarding) {
+        var rowUserName = rowsOnBoarding[row][0];
+        if (rowUserName.toLowerCase() === userName.toLowerCase()) {
+          isUserinOnBoardingSheet = true;
+          break;
+        }
+      }
+
       // If the commit author has signed the CLA, no action is required.
       if (hasUserSignedCla === true) {
         continue;
       }
 
+      if (!isUserinOnBoardingSheet) {
+        userNameNotInClaAndOnBoardingSheet =
+         userNameNotInClaAndOnBoardingSheet + '@' + userName + ',';
+      }
+
       userNamesNotSignedCLA = userNamesNotSignedCLA + '@' + userName + ', ';
     }
 
     if (userNamesNotSignedCLA.length === 0) {
       return;
     }
 
+    if (userNameNotInClaAndOnBoardingSheet.length !== 0) {
+      const commentBody = '@' + teamLeads.onboardingTeam +
+      ' The author of this pull request has not signed CLA.';
+      const commentParam = context.repo({
+        issue_number: prNumber,
+        body: commentBody
+      });
+      await context.github.issues.createComment(commentParam);
```

Comment:
This comment doesn't make sense, what about the case when the contributor is not on the tracker but is on CLA.

Reply:
So sir shall i make oppiabot specific like oppiabot should alert whether which of the following (CLA sheet or onBoarding sheet ) author username is not mentioned. and if username is mentioned in both oppiabot won't alert onBoardingTeam.